docs/sources/reference/commandline/cli.md

```diff
@@ -893,7 +904,6 @@ Creates a new container.
       --cgroup-parent=""         Optional parent cgroup for the container
       --cidfile=""               Write the container ID to the file
       --cpuset-cpus=""           CPUs in which to allow execution (0-3, 0,1)
-      --cpuset-mems=""           Memory nodes (MEMs) in which to allow execution (0-3, 0,1)
```

Comment:
Something weird is going on with your rebase - this edit isn't part of your stuff.
Same for the stuff down below.